src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+module.exports = function(app) {
+  app.use(
+    ['/api', '/auth', '/login', '/metadata'],
```

Comment:
Adding this explicit setupProxy file gives us more control over the proxy behavior with webpack dev server building the react app.

https://create-react-app.dev/docs/proxying-api-requests-in-development/